crates/bindings-csharp/BSATN.Runtime/BSATN/Runtime.cs

```diff
@@ -438,38 +438,3 @@ public AlgebraicType GetAlgebraicType(ITypeRegistrar registrar) =>
     public AlgebraicType GetAlgebraicType(ITypeRegistrar registrar) =>
         enumerable.GetAlgebraicType(registrar);
 }
-
-public readonly struct Dictionary<Key, Value, KeyRW, ValueRW> : IReadWrite<Dictionary<Key, Value>>
```

Comment:
@Centril Actually, can you also remove the relevant branch in BSATN.Codegen as well?

Reply:
I suppose C# tests will yell at you about it too.